### ISSUE TITLE
 scr_cmd:517: FE_DISEQC_SEND_MASTER_CMD failed.

### ISSUE THREAD

**Issue by simonbuehler** [stefantalpalaru/w_scan2#2]
hi,

great work, thanks! i have tried your version but there's a bug also from the original version which  breaks unicable /scr  support:

 `scr_cmd:517: FE_DISEQC_SEND_MASTER_CMD failed.`

the last version http://wirbel.htpc-forum.de/w_scan/w_scan-20170107.tar.bz2 has this fixed, could you merge the fix into your build?

(first fixed version : https://www.vdr-portal.de/forum/index.php?thread/126054-w-scan-findet-bei-unicable-keine-sender/&postID=1237435#post1237435) in case thats better to merge

**Comment by simonbuehler** [comment]
diseqc.c l 574 : should be
```
	if ((err = ioctl(frontend_fd, FE_DISEQC_SEND_MASTER_CMD, diseqc))) {
		info("%s:%d: FE_DISEQC_SEND_MASTER_CMD failed.\n", __FUNCTION__,
		     __LINE__);
		return err;
	}
```

**Comment by stefantalpalaru** [comment]
I merged all the upstream changes up to w_scan-20170107. Thank you for the bug report.
